src/main/kotlin/prison/nomis.kt

```diff
@@ -33,14 +33,15 @@ class NOMIS(model: Model) {
     system.addContainer("NOMIS Web Application",
         "Web Application fronting the NOMIS DB - used by Digital Prison team applications and services",
         "Weblogic App Sever").apply {
-      uses(db, "JDBC")
+      uses(db, "connects to", "JDBC")
     }
 
-    elite2api = system.addContainer("Elite2 API",
+    prisonApi = system.addContainer("Prison API",
         "API over the NOMIS DB used by Digital Prison team applications and services", "Java")
         .apply {
-          setUrl("https://github.com/ministryofjustice/elite2-api")
-          uses(db, "JDBC")
+          addProperty("previous-name", "Elite2 API")
```

Comment:
This use of properties is new; what do you think? I wanted to make the entry findable if someone searches for "Elite2".

Reply:
This sounds sensible. I was looking at properties the other day, and while they're maybe not as suitable for visually distinguishing elements, they look ideal for this use-case. 

Should we use `previous-names` (plural) here, in case a service has been through multiple name changes?

Reply:
> Should we use `previous-names` (plural) here, in case a service has been through multiple name changes?

I would be inclined to do that when the need comes up the first time, what do you think?

Reply:
I think we've got plenty of examples to know it's a use-case. "CNOMIS/PNOMIS/Classic NOMIS, old NOMIS", "new-nomis/elite2/prison-api", "the hub/prisoner content hub". 

Is there any detriment to using `previous-names`: `elite2`? I'd be tempted to go with the more flexible option first...